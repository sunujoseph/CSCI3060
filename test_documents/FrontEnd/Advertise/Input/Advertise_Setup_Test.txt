login
john123smith
advertise
Lenovo Tab M10
!back
advertise
logout