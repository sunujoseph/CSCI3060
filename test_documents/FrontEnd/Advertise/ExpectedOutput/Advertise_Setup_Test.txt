Please log in to the system: 
Please enter your username: 
Enter command: 
Enter Item Name: 
Enter command: 
ERROR: Cannot list further auctions until the next session is completed.
Goodbye!
